app/events/inventory_aggregate.rb

```diff
@@ -9,17 +9,13 @@ def on(*event_types, &block)
     end
 
     # @param organization_id [Integer]
-    # @param first_event [Integer]
-    # @param last_event [Integer]
+    # @param event_time [DateTime]
     # @param validate [Boolean]
     # @return [EventTypes::Inventory]
-    def inventory_for(organization_id, first_event: nil, last_event: nil, validate: false)
+    def inventory_for(organization_id, event_time: nil, validate: false)
       events = Event.for_organization(organization_id)
-      if first_event
-        events = events.where("id >= ?", first_event)
-      end
-      if last_event
-        events = events.where("id <= ?", last_event)
+      if event_time
+        events = events.where("event_time <= ?", event_time)
```

Comment:
I see the `<=` here -- will you need `from_event_time` ... `to_event_time` or `event_time_before`..`event_time_after` or similar?

Reply:
I don't need it right now.  Can't say if I'll need it later?

Reply:
I used this to do my analysis of what was going on with the MHM size 6.  Seems to work as advertised